### PR TITLE
Add DbtDocsGenerateOperator and extend DbtTestOperator 

### PR DESCRIPTION
**Pull request by ayobamshy**
- Extend DbtTestOperator to allow usage of --data (data test) and --schema (schema test)
- Added new DbtDocsGenerateOperator

### CODE REVIEW

File: airflow_dbt/hooks/dbt_hook.py [hunk 1]
```diff
@@ -41,6 +41,8 @@ def __init__(self,
                  dir='.',
                  vars=None,
                  full_refresh=False,
+                 data_test=False,
+                 schema_test=False,
```
Comment (andrewrjones) [inline airflow_dbt/hooks/dbt_hook.py:45]: None of the other variables are suffixed with `_test`, so can we rename them `data` and `schema`?

**Review by andrewrjones — CHANGES_REQUESTED** [review]
Hey, thanks for the contribution! Looks good, just a comment on the variable name, and the lint is currently failing with the following:

```
airflow_dbt/operators/dbt_operator.py:108:111: E501 line too long (112 > 110 characters)
airflow_dbt/operators/dbt_operator.py:109:1: W293 blank line contains whitespace
airflow_dbt/operators/dbt_operator.py:112:1: E302 expected 2 blank lines, found 0
airflow_dbt/hooks/dbt_hook.py:90:1: W293 blank line contains whitespace
```

You can run the linter locally by following the instructions in the README: https://github.com/gocardless/airflow-dbt#code-style

Once these are resolved I'll be happy to merge and release a new version with your changes.

Thanks again!

File: airflow_dbt/hooks/dbt_hook.py (hunk 1 above)
Comment (ayobamshy) [reply airflow_dbt/hooks/dbt_hook.py]: Thanks for taking the time to review Andrew! Changes made in both dbt_hook.py and dbt_operator.py for the variable names and pylint ensures code style adherence.

**Review by andrewrjones — APPROVED** [review]

**Comment by andrewrjones** [comment]
Thanks @ayobamshy!

**Comment by chuhsuanlee** [comment]
Hi I know it's been a while, but just curious if it's possible to have a new version release with DbtDocsGenerateOperator included?

**Pull request merged**